projects/eval_scannet/scannet/scannet_utils.py

```diff
@@ -0,0 +1,85 @@
+# Modified from
+# https://github.com/facebookresearch/votenet/blob/master/scannet/scannet_utils.py
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""Ref: https://github.com/ScanNet/ScanNet/blob/master/BenchmarkScripts
+"""
+
+import csv
+import os
+
+import numpy as np
+from plyfile import PlyData
+
+
+def represents_int(s):
```

Comment:
general utilities like htis could be moved to src/home_robot?

Reply:
i think because this is specifically for the dataset we could also move it to src/home_robot/home_robot/datasets/scannet_utils.py?